### PR TITLE
Update sigma.md

### PR DESCRIPTION
**Pull request by GuillaumeCSekoia**
Updating the documentation to specify the users to put an "exists" value when doing a filter in most cases

### CODE REVIEW

File: docs/xdr/features/detect/sigma.md
Comment (goudyj) [inline docs/xdr/features/detect/sigma.md:120]:
```suggestion
  Now if in that example selection2 is simply `process.parent.name: 'test'` for instance, **Note that you might need to put in selection1 an "exists" value for the field you are excluding**:
```

File: docs/xdr/features/detect/sigma.md
Comment (goudyj) [inline docs/xdr/features/detect/sigma.md:125]:
```suggestion
  Otherwise the rule will also raise an alert if the field `process.parent.name` does not exist. Sometimes it is not an issue, however it can be in case of high false positives without that field for instance.
```

**Review by goudyj — APPROVED** [review]

**Comment by GuillaumeCSekoia** [comment]
Note: waiting for @gaelmuller review before merging just to be 100% sure of what I've written

**Review by gaelmuller — APPROVED** [review]

**Pull request merged**
